packages/shared/src/components/fields/DragDrop.tsx

```diff
@@ -269,22 +263,7 @@ export function DragDrop({
     }
 
     // Call callback with valid files and errors
-    setFilenames(
-      validFiles.map((file: File, index: number) => {
-        const { name } = file;
-
-        if (!renameFileTo) {
-          return name;
-        }
-
-        const chunks = name.split('.');
-        const extension = chunks.pop();
-
-        return validFiles.length === 1
-          ? `${renameFileTo}.${extension}`
-          : `${renameFileTo} (${index + 1}).${extension}`;
-      }),
-    );
+    setFilenames(validFiles.map(({ name }: File) => name));
```

Comment:
We don't show success state now, and just display the popup, so we won't be needing this.